src/command_modules/azure-cli-resource/setup.py

```diff
@@ -33,7 +33,8 @@
 DEPENDENCIES = [
     'azure-mgmt-resource==1.2.1',
     'azure-cli-core',
-    'azure-mgmt-authorization==0.40.0'
+    'azure-mgmt-authorization==0.40.0',
+    'azure-mgmt-managementgroups==2018-03-01-preview'
```

Comment:
Just to be sure we're on the same page `2018-03-01-preview` is a temporary version picked by the SDK CI, actual final version will be `0.1.0`.
So once we're done here, final step will be:
- I release the SDK as 0.1.0 on PyPI
- You do a last commit here that removes the "privates" folder, and change this line to 0.1.0

Reply:
Yes. That's right.